ledger/src/blockstore_db.rs

```diff
@@ -1227,6 +1245,39 @@ impl TypedColumn for columns::OptimisticSlots {
     type Type = blockstore_meta::OptimisticSlotMetaVersioned;
 }
 
+impl Column for columns::MerkleRootMeta {
+    type Index = (Slot, u64);
+
+    fn index(key: &[u8]) -> (Slot, u64) {
+        let slot = BigEndian::read_u64(&key[..8]);
+        let set_index = BigEndian::read_u64(&key[8..]);
+
+        (slot, set_index)
```

Comment:
Can we use `fec_set_index` everywhere instead of `set_index` for consistency with shreds code?